src/XR.tsx

```diff
@@ -48,6 +49,15 @@ export interface XRProps {
    * 1 = maximum foveation, the edges render at lower resolution
    */
   foveation?: number
+  /**
+   * The target framerate for the XRSystem. Smaller rates give more CPU headroom at the cost of responsiveness.
+   * Recommended range is `72`-`120`. Default is unset and left to the device.
+   * @note If your experience cannot effectively reach the target framerate, it will be subject to frame reprojection
+   * which will halve the effective framerate. Choose a conservative estimate that balances responsiveness and
+   * headroom based on your experience.
+   * @see https://developer.mozilla.org/en-US/docs/Web/API/WebXR_Device_API/Rendering#refresh_rate_and_frame_rate
+   */
+  frameRate?: number
```

Comment:
We could write `XRSession.frameRate` here rather than having this `number | undefined` so people could procedurally clamp this down. Could this be undefined or not present on the XRSession though?